engine/flink/avro-util/src/test/scala/pl/touk/nussknacker/engine/avro/KafkaAvroSourceFactorySpec.scala

```diff
@@ -192,9 +193,9 @@ class KafkaAvroSourceFactorySpec extends FunSuite with BeforeAndAfterAll with Ka
   }
 }
 
-class TupleAvroKeyValueDeserializationSchemaFactory[Key, Value](schemaRegistryClientFactory: ConfluentSchemaRegistryClientFactory)
-                                                               (implicit keyTypInfo: TypeInformation[Key], valueTypInfo: TypeInformation[Value])
-  extends ConfluentAvroKeyValueDeserializationSchemaFactory[(Key, Value)](schemaRegistryClientFactory, useSpecificAvroReader = false)(
+class TupleAvroKeyValueKafkaAvroDeserializerSchemaFactoryBase[Key, Value](schemaRegistryClientFactory: ConfluentSchemaRegistryClientFactory)
```

Comment:
Why it is `Base` now? Name is IMO long enough without `Base` :)

Reply:
IDEA Refactor.. ;)